File: codigos-fonte/fibonacci.jl
#Fibonacci para Julia

mutable struct elementos 
	a::Int64
	b::Int64
	ac::Int64
end

function fibonacci(termos)
	valores = elementos(0,1,0)
	
	for i in 0:termos-1
		println(valores.a)
		valores.ac = valores.a + valores.b
		valores.a = valores.b
		valores.b = valores.ac
	end
end

fibonacci(64)

File: codigos-fonte/fibonacci.go
//Fibonacci para Go

package main

import "fmt"

type elementos struct {
	a int
	b int
	ac int
}

func fibonacci(termos int) {
	valores := elementos{a: 0, b: 1, ac: 0}
	for i := 0; i < termos; i++ {
		fmt.Println(valores.a)
		valores.ac = valores.a + valores.b
		valores.a = valores.b
		valores.b = valores.ac
	}
	return
}

func main() {
	fibonacci(64)
}
